### DISCUSSION TITLE
IMPORTANT: Hotfix for polyfill.io security issue

### DISCUSSION THREAD

**Discussion by sarah-beasley** [open-sdg/open-sdg#2117] [Announcements]
It was reported in June 2024 that polyfill.io has been distributing malicious code to its users. Open SDG used javascript resources from this domain so we have removed references to polyfill.io from Open SDG to mitigate the risk of a cyber attack through any websites using the platform. We have replaced the affected code with Cloudflare in line with industry recommendations. 

To mitigate the security risk by removing references to polyfill.io, you will need to point to the hotfix in your _config.yml file in your site repository by changing the line:
`remote_theme: open-sdg/open-sdg@2.3.0`
to
`remote_theme: open-sdg/open-sdg@2.3.2`

If you are not already on version 2.3.0, you can follow the [upgrade instructions](https://open-sdg.readthedocs.io/en/latest/upgrades/upgrading-2-3-0/) to upgrade your site to the latest version, including this hotfix. 

For those not wishing to upgrade fully to 2.3.0 just yet, there is a hotfix for those on version 2.2.0 who will need to change the line:
`remote_theme: open-sdg/open-sdg@2.2.0`
to
`remote_theme: open-sdg/open-sdg@2.2.1`

For those of you that are not on either version 2.3.0 or 2.2.0, you can either update your platform to 2.3.0 following the [instructions](https://open-sdg.readthedocs.io/en/latest/upgrades/upgrading-2-3-0/), or you can override the file by including [this file](https://github.com/open-sdg/open-sdg/blob/2.4.0-dev/_includes/polyfills.html) (named the same, polyfills.html) in your includes folder in your site repository.

Please let us know if you have any issues or questions about doing this.

**Comment by christoph-LSN** [comment]
@sarah-beasley Thank you! It seems everything is working after updating to 2.2.1
